lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -453,7 +453,7 @@ def threshold_state(self, state):
     def needs_monitoring(self):
         """
         Return True if the drive needs to be picked by
-        a Drivemonitor periodic check, False otherwise.
+        a Drive monitor periodic check, False otherwise.
```

Comment:
This should actually be:

    Return True if the drive needs to be picked by the VolumeMonitor
    periodic check, False otherwise.

Reply:
Done :)